manuscript/the-meat.txt

```diff
@@ -1,21 +1,21 @@
 ## Loading data
 
-Great, we’ve got something rendering. But we need to load some data before we can draw a picture. If you still don't have `public/data/h1bs.csv`, now is the time to get it. You can find it on Github, [here](https://github.com/Swizec/h1b-software-salaries), or in the stub project included with the book.
+Great, we’ve got something rendering. Now we need to load some data before we can draw a picture. If you still don't have `public/data/h1bs.csv`, now is the time to get it. You can find it on Github [here](https://github.com/Swizec/h1b-software-salaries), or in the stub project included with the book.
 
-We’re going to use d3.js’s built-in data loading magic and hook it into React’s component lifecycle. 
+We’re going to use d3.js’s built-in data-loading magic and hook it into React’s component lifecycle.
 
 We start by adding three short methods to `H1BGraph`:
 
 {crop-start-line=34,crop-end-line=49,linenos=off,lang=jsx}
 <<[Base methods to load data](code_samples/main.jsx)
 
-React calls `componentWillMount` right before inserting the component into our document. It’s a great opportunity to make any last minute preparations before rendering. In our case, it’s a good spot to start loading data. 
+React calls `componentWillMount` right before inserting the component into our document. It’s a great opportunity to make any last-minute preparations before rendering. In our case, it’s a good spot to start loading data.
 
-I'm sure better spots exist, but loading a 12MB data file is always going to take a couple of seconds. And you should avoid loading data you never intend to render.
+I'm sure better spots exist, but loading a 12MB data file is always going to take a couple of seconds, and you should avoid loading data you never intend to render.
```

Comment:
>> I'm sure better spots exist, but loading a 12MB data file is always going to take a couple of seconds. You should try to avoid loading data you never intend to render.